02/app.js

```diff
@@ -1 +1,12 @@
-console.log('DOM');
\ No newline at end of file
+console.log('DOM');
+
+const linkList = document.querySelectorAll('[data-url]');
+
+if (linkList) {
```

Comment:
Tutaj nie warto sprawdzać bo `querySelectorAll` zwraca tablicę - obojętnie czy to jest pusta tablica czy "uzupełniona" to i tak jest to obiekt, które zawsze jest "prawdziwy".